src/blockstore/store-remote.ts

```diff
@@ -31,12 +31,12 @@ import { SuperThis } from "../types.js";
 // }
 
 export async function RemoteDataStore(sthis: SuperThis, name: string, url: URI, opts: StoreOpts) {
-  const ds = new DataStoreImpl(sthis, name, url, opts);
+  const ds = new DataStoreImpl(sthis, url, opts);
   await ds.start();
   return ds;
 }
 export async function RemoteMetaStore(sthis: SuperThis, name: string, url: URI, opts: StoreOpts) {
-  const ms = new MetaStoreImpl(sthis, name, url, opts /* , true*/);
+  const ms = new MetaStoreImpl(sthis, url, opts);
```

Comment:
will look for the other side of that `true` :)

Reply:
if we need this flags it should be part of storeopts not explicit